.github/workflows/ci.yml

```diff
@@ -2,74 +2,70 @@ name: CI
 on:
   push:
 
+env:
+  GRADLE_OPTS: "-Dorg.gradle.daemon=false"
+
 jobs:
   build:
     strategy:
       matrix:
-        os: [ubuntu-18.04, macos-10.15]
+        os: [ubuntu-20.04, macos-10.15]
     runs-on: ${{ matrix.os }}
 
-    env:
-      GRADLE_ARGS: >-
-        --no-daemon
-        --max-workers 2
-        -Pkotlin.incremental=false
-
     steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-
-      - name: Set up JDK
-        uses: actions/setup-java@v1
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v2
         with:
-          java-version: 1.8
+          distribution: "adopt-hotspot"
```

Comment:
Is Hotspot some special version of Java?

Reply:
From https://adoptopenjdk.net/:

![Screen Shot 2021-06-28 at 10 59 20 AM](https://user-images.githubusercontent.com/98017/123682500-eabfc200-d7ff-11eb-9139-199ede4eb092.png)